library/es_ta_for_indexers.py

```diff
@@ -23,13 +24,34 @@ def create_parser():
 def make_ta_for_indexers(username, password):
     '''
     Splunk_TA_ForIndexers spl generation for ES 4.2.0 and up
+    There are now three versions of ES we're now supporting (changes to makeIndexTimeProperties have been made over different versions).
+    The try/except blocks below are meant to handle the differences in function signature.
     '''
     if not username or not password:
     	raise Exception("Splunk username and password must be defined.")
     sys.path.append(make_splunkhome_path(['etc', 'apps', 'SA-Utils', 'bin']))
     session_key = auth.getSessionKey(username, password)
     from app_maker.make_index_time_properties import makeIndexTimeProperties
+    success = False
     try:
+        spec = {}
+        spec["_app"] = app_info
+        spec["include_indexes"] = include_indexes
+        spec["include_properties"] = include_properties
+        spec.update()
+        archive = makeIndexTimeProperties(spec, session_key)
+        success = True
+    except TypeError:
+        #Some versions have a change that consolidated app_info, namespace, and include_indexes, 
+        #and added include_properties.
+        #Below code is written to handle older versions.
+        pass
+    if success:
+        print archive
```

Comment:
print

Reply:
https://github.com/splunk/splunk-ansible/blob/develop/library/es_ta_for_indexers.py#L41

Reply:
My understanding is that the function is meant to print upon successfully creating the archive? Unless the older one snuck in as well.